lib/passport-configurator.js

```diff
@@ -405,6 +406,26 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
         }
       );
       break;
+    case 'fb-token':
```

Comment:
Shouldn't the generic case `default` work for this has well, since it works for the normal facebook login